services/ui-src/src/components/reports/ReportPageWrapper.tsx

```diff
@@ -42,25 +36,13 @@ export const ReportPageWrapper = () => {
         return (
           <ModalDrawerReportPage route={route as ModalDrawerReportPageShape} />
         );
-      case PageTypes.OVERLAY_MODAL:
-        return <OverlayModalPage route={route as OverlayModalPageShape} />;
       case PageTypes.MODAL_OVERLAY:
         return (
           <ModalOverlayReportPage
             route={route as ModalOverlayReportPageShape}
             setSidebarHidden={setSidebarHidden}
           />
         );
-      case PageTypes.ENTITY_OVERLAY:
-        return (
-          <EntityDetailsOverlay route={route as EntityDetailsOverlayShape} />
-        );
-      case PageTypes.ENTITY_DETAIL_DASHBOARD:
-        return (
-          <EntityDetailsDashboardOverlay
-            route={route as EntityDetailsDashboardOverlayShape}
-          />
-        );
```

Comment:
removed these page types as they are never called directly from the ReportPageWrapper. They are all children of other page types.